release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -845,11 +845,11 @@ This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdmi
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
 
 [id="ocp-4-16-networking-sdn-ovnk-live-migration_{context}"]
-==== Live migration to the OVN-Kubernetes network plugin
+==== Limited live migration to the OVN-Kubernetes network plugin
 
 Previously, when migrating from OpenShift SDN to OVN-Kubernetes, the only available option was an _offline_ migration method. This process included some downtime, during which clusters were unreachable.
 
-This release introduces a _live_ migration method. The live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and {azure-first} Red{nbsp}Hat OpenShift deployment types. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+This release introduces a limited _live_ migration method. The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```

Comment:
```suggestion
This release introduces a limited _live_ migration method. The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}. It is not available for hosted control plane deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
```
In the official docs, we use `hosted control planes` instead of `HyperShift`, with the exception of references to the HyperShift Operator.